src/main/java/ui/MenuControl.java

```diff
@@ -271,6 +272,20 @@ private void onBoardDelete(String boardName) {
             }
         }).collect(Collectors.toList());
     }
+    
+    public void switchBoard() {
+        if (prefs.getLastOpenBoard().isPresent() && prefs.getAllBoards().size() > 1) {
+            List<String> boardNames = new ArrayList<>(prefs.getAllBoards().keySet());
```

Comment:
Do I just have to change ```savedBoards``` in GlobalConfig to LinkedHashMap, or should every access to it be changed accordingly to ensure the order will be retained?